src/HPWH.hh

```diff
@@ -1137,9 +1136,25 @@ class HPWH : public Courier::Sender
     {
         bool overrideFirstHourRating = false;
         FirstHourRating::Desig desig = FirstHourRating::Desig::VerySmall;
+
+        bool overrideAmbientT = false;
+        double ambientT_C = 19.7;
+
     } customTestOptions;
 
-    void makeGeneric(const double targetUEF);
+    struct Fitter;
+    struct GenericOptions;
+    void makeGeneric(const GenericOptions& genericOptions,
+                     StandardTestOptions& standardTestOptions);
+    void makeGenericUEF(double targetEF, double ambientT_C = 19.7);
+    void makeGenericE50(double targetEF) { return makeGenericUEF(targetEF, 10.); }
+    void makeGenericE95(double targetEF) { return makeGenericUEF(targetEF, 35.); }
+    void makeGenericE50_UEF_E95(double targetEF50, double targetUEF, double targetEF95)
```

Comment:
We still need to figure out what to do if targetEF50 or targetEF95 are not provided. Should we adjust them proportionally to the UEF?